erigon-lib/kv/temporal/kv_temporal.go

```diff
@@ -156,18 +156,19 @@ func (db *DB) UpdateNosync(ctx context.Context, f func(tx kv.RwTx) error) error
 type Tx struct {
```

Comment:
For this struct consider changing to a less generic name, like say `TemporalTx` later. (Unrelated to this PR)